apps/notification/models.py

```diff
@@ -73,3 +75,37 @@ class Meta:
     @staticmethod
     def get_for(user):
         return Notification.objects.filter(receiver=user).distinct()
+
+
+class Assignment(models.Model):
+    """
+    Assignment Model
+    """
+    created_at = models.DateTimeField(auto_now=True)
+    created_by = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        related_name='created_by',
+    )
+    created_for = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        related_name='created_for',
+    )
+    project = models.ForeignKey(
+        Project,
+        on_delete=models.CASCADE,
+        blank=True,
```

Comment:
`blank=True` is not required here. Removing this will create another migration again, so revert back and create a single migration.

Ref: https://docs.djangoproject.com/en/3.1/ref/models/fields/#blank